apps/music/views/playlist-detail/view.js

```diff
@@ -38,10 +38,14 @@ PlaylistDetailView.prototype.render = function() {
 };
 
 PlaylistDetailView.prototype.getPlaylist = function() {
+  var unpaddedIndex = IntlHelper.get('unpaddedIndex');
+
   return this.fetch('/api/playlists/info/' + this.params.id)
     .then(response => response.json())
     .then(songs => {
-      songs.forEach((song, index) => song.index = index + 1);
+      songs.forEach((song, index) => {
+        song.index = unpaddedIndex.format(index + 1);
```

Comment:
is `song.index` only used for displaying? If so, you may want to move it closer to where the DOM injections happen. If it's used for anything else (like, constructing class/id names) then you don't want to format it because it becomes a string (and your class would be 'song-title-١`)